implementation/src/main/java/io/smallrye/jwt/auth/principal/KeyLocationResolver.java

```diff
@@ -213,21 +193,80 @@ private void loadContents() throws IOException {
         }
 
         if (is == null) {
-            throw new IOException("No resource with the named " + authContextInfo.getPublicKeyLocation() + " location exists");
+            throw new IOException("No resource with the named " + keyLocation + " location exists");
```

Comment:
I'd suggest `throw new IOException("No resource exists at " + keyLocation);`.

Reply:
Sure will do, a user provided this message so I thought of keeping it as it was